packages/test-qna/tests/delivery-date.test.js

```diff
@@ -15,15 +15,11 @@
  */
 /* eslint prefer-arrow-callback: "off" */
 
-const sinon = require('sinon');
```

Comment:
I think you can remove the dev dep on sinon in the package.

Reply:
ok